### DISCUSSION TITLE
Stuck at logging in...

### DISCUSSION THREAD

**Discussion by DunklerPhoenix** [9p4/jellyfin-plugin-sso#222] [Q&A] [answered]
Heho
I have the problem that I stuck at "Logging in".
The browser blocks the iFrame because of mixed content.

`Mixed Content: The page at 'https://jellyfin.domain.nope/sso/OID/redirect/authentik?code=a385e1a3437e7&state=dTAqD5TR4H0g' was loaded over HTTPS, but requested an insecure frame 'http://jellyfin.domain.nope/web/index.html'. This request has been blocked; the content must be served over HTTPS.`

How can I fix this? 
I already tried to set the firewall setting to 'all=https://jellyfin.domain.nope' but it doesn't help.

Greetings 
Phoenix

**Comment by DunklerPhoenix** [comment]
Hm Jellyfin is behind a reverse proxy, but I need to activate the https port for jellyfin and use this port in the reverse proxy. Otherwise it doesn't work. It would be good if we can set the protocol in the sso configurations. :D

**Reply by 9p4** [reply, marked as answer]
You can set it to override to https in the plugin config with the override text box set to https.

**Reply by DunklerPhoenix** [reply]
Ohhhhhhhhh ups 😅

**Comment by Graxo** [comment]
I have the same issue on mobile. It works fine on my desktop/laptop. But not on my phone, both iOS and Android have issues. Its just stuck on `Logging in...`. I have the Scheme Override set to `https`.
